### ISSUE TITLE
[Bug]: No sound and crash on close.

### ISSUE THREAD

**Issue by alocke8181** [AllTheMods/ATM-10#1167]
### Possible Fixes

Yes

### Modpack Version

1.20

### What happened?

NeoForge version - 21.1.79
Cursed Forge version - 1.265.0
Minecraft version - 1.21.1

The game has been launching with no sound. Only the narrator has been working. Minecraft does appear on my Windows Volume Mixer, and there are no issues with other program sounds on my computer. 

I've tried reinstalling ATM 10 multiple times, as well as making sure everything is on the latest versions. Whenever I close Minecraft, the launcher appears and says the game crashed with Exit Code -1073740791. I have not installed any additional mods to the pack. 

### latest.log

https://gist.github.com/alocke8181/7a8f9703d2425c535bb9e8ce34dadcf7

**Comment by Seanypat** [comment]
I don't have the crashing issue but I do have the sound issue. I too noticed no sounds for the menu or the game about the time I updated to 1.20. I still have an instance of ATM9 and its sounds work fine, so it's only ATM10 apparently.

**Comment by SamTheLegendMan** [comment]
I just want to contribute to this, I installed ATM10 through the curseforge launcher today, and have the exact same bug, ive tried everything from reloading sounds and resource packs, adjusting system devices and volumes, none of it works, I saw another thread where they had similar issues, but said it must've been something locally for them, I dont think this is the case, gotta be something with the download as i only downloaded today.

**Comment by Seanypat** [comment]
I have my laptop connected to a large flat screen TV via HDMI. The TV is connected to a separate sound system. The only thing from my laptop that doesn't have sound is ATM10. The video is fine. I've tested the different sound settings in Minecraft on my laptop and the only time sound is produced is when I set it to the laptop speakers.

Do either of you have a similar setup - HDMI from computer to external display connected to a sound system?

**Comment by SamTheLegendMan** [comment]
Yeah, im playing on my Desktop, my monitor is connected via DVI-D not HDMI, and im using a wireless headset connected via bluetooth & USB, changing it to wired mode has no effect either.

**Comment by SamTheLegendMan** [comment]
Okay... so just an update, after relaunching for the 7th time, sound has suddenly returned and is working, i didnt do anything different, im not entirely sure what changed....

**Comment by Seanypat** [comment]
I'm jealous but glad it's working for you. Hopefully mine will just start working again soon.

**Comment by alocke8181** [comment]
Okay so a super weird solution I've found is to have another program running that's producing sound. No idea how or why it works, but it seems to be consistently true. Could someone else try it to verify?

**Comment by Giblejimmy** [comment]
As the devs have still yet to comment on this bug, and it just happened to me, I can confirm that for some reason the modpack's sound will not function at all upon opening the pack on my end unless I have literally ANYTHING making detectable sound on my computer.

Throw down a Youtube video for a few minutes, hit play on the modpack, BOOM sound will work. Working on something in silence, hit play, no sound at all and no way to correct without restarting the pack. This is legitimately the only modpack out of the last 40 I have played to date to ever have this problem which means any audio-focused mods that may be installed (I didn't read the list) may be the culprit.

So if you're like me and googled "All the Mods 10 no sound" or "ATM10 no sound" your fix is to just play a 5-10 minute Youtube video while your pack warms up.

**Comment by Cannonfodderp** [comment]
Boys I think I've found it. Before, I kept getting errors with OpenAL not initializing and disabling the sound in the game. I found on another thread that **disabling all other playback devices except the one you wish to use** is the way to go and it worked. 
I have a ton of virtual audio cables and channels I didn't use and apparently the game didn't like that and just shit itself. Right now I'm using VoiceMeeter and the only playback devices I have enabled are the channel I use to play audio and my headphones as a device enabled. Hopefully this helps some of you!
